packages/epetra/test/VbrMatrix/cxx_main.cpp

```diff
@@ -1740,7 +1740,7 @@ int checkMergeRedundantEntries(Epetra_Comm& comm, bool verbose)
   int localProc = comm.MyPID();
 
   int myFirstRow = localProc*3;
-  int myLastRow = myFirstRow+2;
+  unsigned int myLastRow = myFirstRow+2;
```

Comment:
Wait, woah, why is this `unsigned` here?

Reply:
This took care of the Werror coming from lines 1758 & 1897:
`error: assuming signed overflow does not occur when assuming that (X + c) < X is always false [-Werror=strict-overflow]`